packages/desktop/src/renderer/components/widgets/channels/NestedMessageContent.tsx

```diff
@@ -34,10 +43,19 @@ export interface NestedMessageContentProps {
 export const NestedMessageContent: React.FC<NestedMessageContentProps> = ({
   message,
   pending,
+  openUrl,
   uploadedFileModal
 }) => {
   const classes = useStyles({})
 
+  const componentDecorator = (decoratedHref: string, decoratedText: string, key: number): ReactNode => {
+    return (
+      <a onClick={() => { openUrl(decoratedHref) }} className={classNames({ [classes.link]: true })} key={key}>
+        {decoratedText}
+      </a>
+    )
+  }
+
   return (
     <Grid item>
       {message.type === 2 ? ( // 2 stands for MessageType.Image (cypress tests incompatibility with enums)
```

Comment:
were is case for message.type 4? and for image size from my last changes here?

Reply:
never mind its compered with master

Reply:
This PR points to master, not develop xD

Reply:
Develop is not yet merged